app/ParserTests/ParserTests.hs

```diff
@@ -6,12 +6,77 @@ Module containing test cases for Requirement Parsers.
 -}
 
 module ParserTests.ParserTests
-( reqTestSuite ) where
+(  reqTestSuite  ) where
 
-import Test.HUnit
+import qualified Text.Parsec as Parsec
+import Text.Parsec.String (Parser)
+import Database.Requirement
+import WebParsing.ReqParser
+import Test.HUnit ( assertEqual, Test(..) )
 
-test1 :: Test
-test1 = TestCase (assertEqual "TEST PASS" ("True") ("True"))
+-- Function to facilitate test case creation given a string, Req tuple
+createTest :: (Eq a, Show a) => Parser a -> (String, a) -> Test
+createTest parser (input, expected) = let courseReq =  Parsec.parse parser "" input
+                                      in TestCase $ (assertEqual ("for (" ++ input ++ "),") (Right expected) courseReq)
 
+-- categoryParser Tests
+orInputs :: [(String, Req)]
+orInputs = [("CSC120H1/CSC148H1", OR [J "CSC120H1", J "CSC148H1"])
```

Comment:
![image](https://user-images.githubusercontent.com/21252090/28242538-bc51091a-697c-11e7-91ad-3ff18a0ed8ff.png)

https://github.com/tibbe/haskell-style-guide/blob/master/haskell-style.md

I will update the formatting as you requested, but I was wondering if we should adopt the guidelines in this link for the future.